debian/tests/integrationtests

```diff
@@ -14,6 +14,7 @@ if [ -z "$SNAPCRAFT_SLOW_TESTS" ]; then
     SNAPCRAFT_SLOW_TESTS=1
 fi
 
+mkdir -p /home/ubuntu/autopkgtest_tmp
 for suite in $suites; do
-    su ubuntu -c "SNAPCRAFT_FROM_INSTALLED=1 TEST_STORE=fake ADT_TEST=1 PATH=/snap/bin:$PATH python3 -m unittest discover -b -v -s ${suite}"
+    su ubuntu -c "SNAPCRAFT_FROM_INSTALLED=1 TEST_STORE=fake ADT_TEST=1 PATH=/snap/bin:$PATH TMPDIR=/home/ubuntu/autopkgtest_tmp python3 -m unittest discover -b -v -s ${suite}"
```

Comment:
Can we not use $HOME here? I guess we're `su`ing as `ubuntu` anyway, but this hard-coded path still bothers me a little.

Reply:
@kyrofa how do I make sure that $HOME is not expanded to /root?

I tried this, which made me hard-code it:

```
# su ubuntu -c "TMPDIR=$HOME/autopkgtest_tmp python3 -c \"import os; print(os.environ.get('TMPDIR'))\""
/root/autopkgtest_tmp
```

Reply:
Ohh, this works:
```
# su ubuntu -c "TMPDIR=\$HOME/autopkgtest_tmp python3 -c \"import os; print(os.environ.get('TMPDIR'))\""
/home/ubuntu/autopkgtest_tmp
```
I'll update it.